experiment/coverage/apicoverage_test.go

```diff
@@ -0,0 +1,197 @@
+			Negative: false,
+			Reg:      reAlphaAPI,
+			apisOpenapi: apiArray{
+				apiData{Method: "GET", URL: "/apis/resources/v1/"},
+				apiData{Method: "POST", URL: "/apis/resources/v1/"},
+				apiData{Method: "GET", URL: "/apis/resources/v2alpha1/"},
+				apiData{Method: "POST", URL: "/apis/resources/v2alpha1/"},
+				apiData{Method: "GET", URL: "/apis/resources/v1beta1/"},
+				apiData{Method: "POST", URL: "/apis/resources/v1beta1/"},
+			},
+			apisTested: apiArray{
+				apiData{Method: "GET", URL: "/apis/resources/v1/"},
+				apiData{Method: "GET", URL: "/apis/resources/v2alpha1/"},
+				apiData{Method: "GET", URL: "/apis/resources/v1beta1/"},
+			},
+			ExpectedTested: apiArray{
+				apiData{Method: "GET", URL: "/apis/resources/v2alpha1/"},
+			},
+			ExpectedAll: apiArray{
+				apiData{Method: "GET", URL: "/apis/resources/v2alpha1/"},
+				apiData{Method: "POST", URL: "/apis/resources/v2alpha1/"},
+			},
+		},
+		{
+			//Test Beta APIs are returned
+			Negative: false,
+			Reg:      reBetaAPI,
+			apisOpenapi: apiArray{
+				apiData{Method: "GET", URL: "/apis/resources/v1/"},
+				apiData{Method: "POST", URL: "/apis/resources/v1/"},
+				apiData{Method: "GET", URL: "/apis/resources/v2alpha1/"},
+				apiData{Method: "POST", URL: "/apis/resources/v2alpha1/"},
+				apiData{Method: "GET", URL: "/apis/resources/v1beta1/"},
+				apiData{Method: "POST", URL: "/apis/resources/v1beta1/"},
+			},
+			apisTested: apiArray{
+				apiData{Method: "GET", URL: "/apis/resources/v1/"},
+				apiData{Method: "GET", URL: "/apis/resources/v2alpha1/"},
+				apiData{Method: "GET", URL: "/apis/resources/v1beta1/"},
+			},
+			ExpectedTested: apiArray{
+				apiData{Method: "GET", URL: "/apis/resources/v1beta1/"},
+			},
+			ExpectedAll: apiArray{
+				apiData{Method: "GET", URL: "/apis/resources/v1beta1/"},
+				apiData{Method: "POST", URL: "/apis/resources/v1beta1/"},
+			},
+		},
+		{
+			//Test Stable APIs are returned
+			Negative: true,
+			Reg:      reNotStableAPI,
+			apisOpenapi: apiArray{
+				apiData{Method: "GET", URL: "/apis/resources/v1/"},
+				apiData{Method: "POST", URL: "/apis/resources/v1/"},
+				apiData{Method: "GET", URL: "/apis/resources/v2alpha1/"},
+				apiData{Method: "POST", URL: "/apis/resources/v2alpha1/"},
+				apiData{Method: "GET", URL: "/apis/resources/v1beta1/"},
+				apiData{Method: "POST", URL: "/apis/resources/v1beta1/"},
+			},
+			apisTested: apiArray{
+				apiData{Method: "GET", URL: "/apis/resources/v1/"},
+				apiData{Method: "GET", URL: "/apis/resources/v2alpha1/"},
+				apiData{Method: "GET", URL: "/apis/resources/v1beta1/"},
+			},
+			ExpectedTested: apiArray{
+				apiData{Method: "GET", URL: "/apis/resources/v1/"},
+			},
+			ExpectedAll: apiArray{
+				apiData{Method: "GET", URL: "/apis/resources/v1/"},
+				apiData{Method: "POST", URL: "/apis/resources/v1/"},
+			},
+		},
+	}
+	for _, test := range testCases {
+		resTested, resAll := getTestedAPIsByLevel(test.Negative, test.Reg, test.apisOpenapi, test.apisTested)
```

Comment:
BTW, In `parseAPILog()` it would be better to use `url.Parse()` instead of `strings.Split()`

Reply:
++, I will change it with the above unit test to verify the behavior.